lib/appmetrics-prometheus.js

```diff
@@ -273,6 +262,267 @@ function stringHttpRequestDuration(httpURLDataList) {
   return content;
 }
 
+function stringProcessUptime() {
+  return [
+    '# HELP process_uptime_count_seconds The number of seconds for which the current Node.js process has been running',
+    '# TYPE process_uptime_count_seconds counter',
+    `process_uptime_count_seconds ${process.uptime()}\n`,
+  ];
+}
+
+function stringLoopData(latestLoopEvent) {
+  if (!latestLoopEvent) return;
+  return [
+    '# HELP event_loop_tick_min_milliseconds The shortest tick time in the event loop samples, in milliseconds',
+    '# TYPE event_loop_tick_min_milliseconds guage',
+    `event_loop_tick_min_milliseconds ${latestLoopEvent.minimum}`,
+
+    '# HELP event_loop_tick_max_milliseconds The longest tick time in the event loop samples, in milliseconds',
+    '# TYPE event_loop_tick_max_milliseconds guage',
+    `event_loop_tick_max_milliseconds ${latestLoopEvent.maximum}`,
+
+    '# HELP event_loop_tick_count The number of event loop ticks in the last interval',
+    '# TYPE event_loop_tick_count counter',
+    `event_loop_tick_count ${latestLoopEvent.count}`,
+
+    '# HELP event_loop_tick_average_milliseconds The average tick time in the event loop samples, in milliseconds',
+    '# TYPE event_loop_tick_average_milliseconds guage',
+    `event_loop_tick_average_milliseconds ${latestLoopEvent.average}`,
+
+    '# HELP event_loop_cpu_user The percentage of 1 CPU used by the event loop thread in user code the last interval. This is a value between 0.0 and 1.0.',
+    '# TYPE event_loop_cpu_user guage',
+    `event_loop_cpu_user ${latestLoopEvent.cpu_user}`,
+
+    '# HELP event_loop_cpu_system The percentage of 1 CPU used by the event loop thread in system code in the last interval. This is a value between 0.0 and 1.0.',
+    '# TYPE event_loop_cpu_system guage',
+    `event_loop_cpu_system ${latestLoopEvent.cpu_system}\n`,
+  ].join('\n');
+}
+
+function stringGCData(latestGCEvent, maxHeapUsed, gcDurationTotal) {
```

Comment:
units should all be seconds, names should all start with `nodejs_`, and consider using the same names as https://github.com/SimenB/node-prometheus-gc-stats#metrics-exposed if possible, I suspect the info comes from the same underlying V8 C++ APIs. Also, "guage".